core/tests/loadnext/src/config.rs

```diff
@@ -166,7 +166,7 @@ fn default_accounts_amount() -> usize {
 }
 
 fn default_duration_sec() -> u64 {
-    let result = 300;
+    let result = 400;
```

Comment:
Loadtest results seem to be high enough again, does it make sense to return 300?

Reply:
lets try (i have no idea why though?)